app/controllers/rooms_controller.rb

```diff
@@ -6,8 +6,15 @@ class RoomsController < ApplicationController
 
   # GET /rooms/:public_id
   def show
-    @jams = @room.jams.order(created_at: :desc).last(20).to_a
-    @current_jam = @jams.shift
+    jams = @room.jams.order(created_at: :desc).last(20).to_a
```

Comment:
`@room.jams.order(created_at: :desc).last(20).to_a` == `first(20)` which is not what you want. The simplest is `@room.jams.last(20).reverse` -- (no `.to_a`).